etc/update-latest-release.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import os
+
+from datetime import datetime
+from github import Github, Auth
+
+API_KEY = os.getenv("API_KEY")
+if API_KEY == None:
+    print("API key was not found! Authentication will fail!\nSet the environment variable API_KEY "
+          "to a github access token and try again!\n")
+
+assert API_KEY != None
```

Comment:
It seems weird to catch an error, print a nice error message, but then proceed and instead rely on an `assert` to throw an error?
```suggestion
    print("API key was not found! Authentication will fail!\nSet the environment variable API_KEY "
          "to a github access token and try again!\n")
    sys.exit(1)
```

Reply:
`sys.exit(1)` (and also a failing assert) will stop the update.sh script.

Should we use the builtin `exit()`, so that we exit from the python script, but the update script keeps going ?

Reply:
Fine by me